common/src/main/resources/breedingDefault.json

```diff
@@ -60,14 +60,14 @@
 		"random": "above 45"
 	  }
 	],
-	"BLUE": [
+	"GREEN": [
 	  {
 		"childColor": "WHITE",
 		"conditions": "bothParentsFedGold",
 		"random": "above 60"
 	  },
 	  {
-		"childColor": "BLUE",
+		"childColor": "GREEN",
```

Comment:
This change is not directly related to issue.

It seems more nature to change it to GREEN.